NSS-Spontaneous-Spoonbills/Sprint3
# Cr migration of models

## Link to Ticket

Closes #24 

## Steps to Test

```sh
git fetch --all
git checkout cr_migrationOfModels
python manage.py makemigrations
python manage.py migrate
python manage.py runserver
```

1. Open db browsers and double check all models have made it in there correctly
1. In server register yourself as a user
1. Should see username and welcome page
1. Logout
1. Login with incorrect information
1. You should be shown an incorrect credentials display
1. Login with the information you used to register
1. Should sign you back in and display your username
1. Confirm your user information is in the db

## Mentions @username

@jessnswift @obscure-shadow @MrErin @JacobAndrewSmith92 

## Testing
